src/System/Console/GetOpt/Generics/Modifier.hs

```diff
@@ -24,6 +28,32 @@ data Modifier
     --   through the @fieldName@ by @customName@.
   deriving (Show, Eq, Ord)
 
+data Modifiers = Modifiers {
+  _shortOptions :: M.Map String [Char],
```

Comment:
`ghc` complained about unused identifiers. But I still want to have them for documentation.